tests/behat/serve-bootstrap.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+use SilverStripe\CMS\Model\SiteTree;
```

Comment:
I don't think this coupling to CMS makes sense; we probably need a `serve-bootstrap.php` within the CMS module itself.